Задания с 18.02.2022/Task2/Task2/BWT.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+public class BWT
```

Comment:
```suggestion
public static class BWT
```
И комментарии было бы неплохо